.github/workflows/ci-weekly.yml

```diff
@@ -1,9 +1,10 @@
-name: FAST Workflows - Cross Platforms
+name: CI - Cross Platform Validation
 
 on:
   schedule:
-  - cron: '0 0 * * 3'
-  
+  - cron: 0 7 * * 3 # Every Wednesday at 7am

```

Comment:
Are you sure you should remove the quotes? The documentation has them in: https://help.github.com/en/actions/configuring-and-managing-workflows/configuring-a-workflow

Is this potentially an indenting issue? I see the cron line might need to be indented one step further.

Reply:
The spacing is not an issue as you can see this is the pattern that's working on ci-daily.yml currently.  I have found at least two other sources of documentation not using quotes that I wanted to give a try.

Reply:
Ideally, it would choose to run by 7am tomorrow just being in a PR. But, I think it has to be committed.  I'll find this answer out tomorrow as I won't commit until after 7am just to see.  This functionally is quite a bit different than ADO so I have a little to figure out still